server/queries/class/deleteStudentQuery.ts

```diff
@@ -0,0 +1,24 @@
+import { Op } from 'sequelize';
+import { ClassStudent } from '../../models';
+
+const deleteStudentQuery = async (classId: string, studentId: string) => {
```

Comment:
Let it be more clear name, you are just removing a student from the class, and the same for your controller.